luminoth/models/fasterrcnn/fasterrcnn_test.py

```diff
@@ -146,6 +146,23 @@ def _gen_anchors(self, config, feature_map_shape):
             results = sess.run(results)
             return results
 
+    def _get_losses(self, prediction_dict):
+        image = tf.placeholder(
+            tf.float32, shape=self.image.shape)
+        gt_boxes = tf.placeholder(
+            tf.float32, shape=self.gt_boxes.shape)
+        model = FasterRCNN(self.config)
+        results = model(image, gt_boxes)
```

Comment:
Do we need to run the results?

Reply:
I think we do so the rpn and rcnn objects of the model are initialized.